docs/ballerina-by-example/examples/base-path-and-path/base-path-and-path.bal

```diff
@@ -1,23 +1,23 @@
 import ballerina/http;
 import ballerina/mime;
 
-@Description {value:"Attributes associated with the service endpoint is defined here."}
+@Description {value:"Define the attributes associated with a service's endpoint."}
```

Comment:
service endpoint is a concept and more clear. Shall we leave that, instead of saying service's endpoint?

Reply:
Changed to "service endpoint".